### PR TITLE
New site launch

### PR DESCRIPTION
**Pull request by ewels**
Do not merge until we are ready to go live with the new site.

### CODE REVIEW

**Comment by ewels** [comment]
Apologies if some of this is underway, but figured it was easiest to make a list on the PR. I'm using checkboxes, hopefully you can tick them off as they're done ✅ 

You're working on Program and Speakers currently, so I've not looked at them 🙈 😁 

## Header / Footer

1. [ ] Footer should be fixed to the bottom of the page, see this Figma preview:

https://user-images.githubusercontent.com/465550/177201751-99ffd4bd-7b18-4962-8036-22bf8f31c683.mov

2. [ ] Social icons in bottom right of the footer are missing
3. [ ] `© 2022 Seqera Labs` should link to `https://seqera.io`
4. [ ] Header links should have active state (green) when on that page.
5. [ ] No navigation present on mobile

## Homepage

1. [ ] No background gradient
2. [ ] Scrolling _"Call for abstracts now open."_ text should link to `call-for-abstracts/`
3. [ ] Nav should be transparent on this page only
4. [ ] Key Information > Venue: Missing map pin icon
5. [ ] Key Information > Streaming: Old incorrect content + missing icons
6. [ ] Key Information: Can we drop the `1rem` top-margin for the bullet points?

## Call for abstracts

1. [ ] Scrolling _"Call for abstracts now open."_ text needs gradient background (no link, as this is the target page)
2. [ ] Images have no rounded corners
3. [ ] Background for images have too much rounding - `12px` but should be `4px`
4. [ ] Images extend beyond page width, need overflow hiding (right hand side only, left is fine)
    image
5. [ ] _"Apply now"_ button should have external-link icon added

## Call for abstracts > Instructions

1. [ ] Back icon is missing <kbd><</kbd> icon
2. [ ] Both panels missing the _"Apply now"_ text link at the bottom (instructions PDF not ready so that link in Figma can be ignored)
    * _"Apply now"_ text link should link back to `call-for-abstracts/`
4. [ ] _"View Abstracts"_ button - I expected some hover state, maybe green background?
5. [ ] I'm pretty sure that the title should be white. It's green in Figma, I've left a comment to ask, but for now change it to white 👍🏻 

## Venue

1. [ ] Image overlaps right hand edge of screen, as in _Call for abstracts_
2. [ ] Scrolling _"Call for abstracts now open."_ needs gradient background and link to `call-for-abstracts/`
3. [ ] Google Maps API needs fixing
4. [ ] Missing map pin icon after address line

## Register

1. [ ] External link icon for the _Register_ button
2. [ ] Typo in fees > virtual: `aNextflow` should be `Nextflow` (was in Figma too, just corrected there)

**Comment by christopher-hakkaart** [comment]
Looks really good!

Only issue I found
- [ ] Home page has extra scroll space. Video attached.

https://user-images.githubusercontent.com/76951771/177349613-dca8a9d8-93a2-441c-a87c-247028bc9ca9.mov

File: src/layout/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          <div className="border-white border-t opacity-40" />
+          <div className="flex justify-between py-4">
+            <div>
+              <Link to="https://seqera.io/" noBorder className="typo-small">
+                &copy; 2022 Seqera Labs
+              </Link>
+            </div>
+            <div className="flex items-center md:justify-end">
+              <div className="mr-4">
+                <Link to="http://twitter.com/hashtag/NextflowSummit" noBorder>
```
Comment (ewels) [inline src/layout/Footer.jsx:24]: This is bizarre and I don't understand why, but without the `?src` query string I get no results on the twitter link. Adding this seems to fix it.. 🤷🏻‍♂️ 

```suggestion
                <Link to="https://twitter.com/hashtag/NextflowSummit?src=hashtag_click" noBorder>
```

File: src/pages/program/nf-core-hackathon.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                  9 AM, Oct 10 - 5 PM, Oct 12, 2022
+                </p>
+                <h3 className="typo-h4 mb-4">
+                  Start - Finish
+                </h3>
+                <p className="typo-body">
+                  The nf-core hackathon will be held before the Nextflow Summit, from 10-12 Oct, 2022.
+                </p>
+                <p className="typo-body mb-4">
+                  <Link to="/" className="text-green-600">
```
Comment (ewels) [inline src/pages/program/nf-core-hackathon.jsx:66]:
```suggestion
                  <Link to="https://nf-co.re/events/2022/hackathon-october-2022" className="text-green-600">
```

File: src/pages/program/nf-core-hackathon.jsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                    Visit the nf-core hackathon event page
+                  </Link>
+                  {' '}
+                  to learn more.
+                </p>
+                <p className="typo-body">
+                  <Link to="">
```
Comment (ewels) [inline src/pages/program/nf-core-hackathon.jsx:73]:
```suggestion
                  <Link to="https://goo.gl/maps/K3chvdYLa9BfDpaD9">
```

File: src/pages/register.jsx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+            <div className="col-full md:col-6 mt-4 md:mt-0">
+              <div className="bg-black border border-gray-800 rounded-md p-8 shadow-xl relative">
+                <p className="typo-intro text-green-600">
+                  Oct 12-14
+                </p>
+                <h3 className="typo-h4 mt-6">
+                  Nextflow Summit 2022
+                </h3>
+                <p className="typo-body mt-6">
+                  Join the nf-core hackathon held in Barcelona and virtually.
```
Comment (ewels) [inline src/pages/register.jsx:57]:
```suggestion
                  Join the Nextflow Summit held in Barcelona and virtually.
```

**Review by ewels — COMMENTED** [review]
Some minor copy / link errors, see suggested changes for fixes 👇🏻

**Comment by ewels** [comment]
Updated condensed list of remaining points from https://github.com/nextflow-io/summit-website/pull/1#issuecomment-1174060204:

## Header / Footer

1. [ ] Footer should be sticky, fixed to the bottom of the page
2. [ ] Scrolling _"Call for abstracts now open"_ should also be sticky + fixed to bottom of the page (above footer)
3. [ ] Scrolling _"Call for abstracts now open"_ needs colour gradient background (all pages except homepage)
6. [ ] Homepage -> Key Information: Can we drop the `1rem` top-margin for the bullet points?
5. [ ] External link icons for _"Apply now"_ (call for abstracts page main button) / _"Register"_ button (register main button)

File: src/pages/program/oct-14.jsx [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+                    <Link to="/" className="typo-body text-gray-600">
+                      Watch on youtube
+                    </Link>
+                  </span>
+                </div>
+              </div>
+
+              <div className="bg-black border border-gray-800 px-4 py-6 lg:p-8 rounded-md shadow-xl mt-4">
+                <p className="typo-intro text-green-600 mb-4">
+                  12 AM (20 min)
```
Comment (ewels) [inline src/pages/program/oct-14.jsx:156]:
```suggestion
                  1:20 - 2:20 PM (60 min)
```

File: src/pages/index.jsx [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+              >
+                <Accordion.Summary>
+                  <span className="typo-intro">
+                    Key dates
+                  </span>
+                </Accordion.Summary>
+                <Accordion.Detail>
+                  <List type="bullet" iconClassName="text-white">
+                    <List.Item className="typo-body">
+                      July 1: Registration for the Nextflow Summit opens
```
Comment (ewels) [inline src/pages/index.jsx:123]:
```suggestion
                      July 7: Registration for the Nextflow Summit opens
```

File: src/pages/call-for-abstracts/index.jsx [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+      <div className="py-32 bg-gray-900 text-white">
+        <div className="container-lg text-center">
+          <h1 className="typo-h2 mb-4">
+            Call for abstracts open
+          </h1>
+          <p className="typo-body max-w-md mb-8 mx-auto">
+            Submit your talk or poster for one of the key themes: Nextflow, community, ecosystem, or software.
+          </p>
+          <p className="typo-body">
+            Now - July 22, 2022: Call for abstracts
```
Comment (ewels) [inline src/pages/call-for-abstracts/index.jsx:46]:
```suggestion
            June 17 - July 22, 2022: Call for abstracts
```

File: src/pages/program/oct-14.jsx [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+                Fri, Oct 14
+              </Tabs.Item>
+            </Tabs>
+          </div>
+          <div className="row mt-4">
+            <div className="col-full lg:col-9">
+              <div className="bg-black border border-gray-800 px-4 py-6 lg:p-8 rounded-md shadow-xl">
+                <div className="flex items-center justify-between mb-4">
+                  <p className="typo-intro text-green-600">
+                    11 - 12 AM (60 min)
```
Comment (ewels) [inline src/pages/program/oct-14.jsx:64]:
```suggestion
                    11 AM - 12 PM (60 min)
```

**Pull request merged**